resources/cert-manager/README.md

```diff
@@ -0,0 +1,61 @@
+# cert-manager
+
+## Overview
+
+`cert-manager` is a version of [Jetstack's cert-manager](https://cert-manager.io) customized to Kyma's needs.
+
+Quoting from the [official README](https://github.com/jetstack/cert-manager):
+
+>cert-manager is a Kubernetes addon to automate the management and issuance of
+>TLS certificates from various issuing sources.
+>
+>It will ensure certificates are valid and up to date periodically, and attempt
+>to renew certificates at an appropriate time before expiry.
+
+
+For more information check [links](#links) section.
+
+## Prerequisites
+
+- Kubernetes 1.11+
+
+## Details
+
+### Changes introduced
+
+The `crds.yaml` file has been split into multiple files. Each of these contains exactly one CRD with its name indicating the resource type. They are available in the `files/` directory.
+
+### Installation
+
+To install `cert-manager`, use the Kyma Installer and follow these steps:
+
+1. Manually apply CRDs from the `files/` directory:
+
+   ```bash
+   # Kubernetes 1.15+
```

Comment:
I just noticed that. In prerequisites you have Kubernetes 1.11+, but in the instructions for the **only** way to install it, you have Kubernetes 1.15+? 
I'm not sure about this...

Reply:
We have agreed to remove `Prerequisites` section entirely.